ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -228,7 +228,7 @@ const TopologyComponent = ({
 
     function resetGraphToDefault() {
         removeExtraneousEdges();
-        controller.fromModel(model);
+        controller.fromModel(model, true);
```

Comment:
What does the `true` do?

Reply:
it makes it so that the new model is merged w the previously existing model. i think it's `true` by default but i just wanted to explicitly put it there